e2e/testcases/multi_sync_test.go

```diff
@@ -120,7 +120,9 @@ func TestMultiSyncs_Unstructured_MixedControl(t *testing.T) {
 	newRepos = append(newRepos, nn5)
 
 	if nt.GitProvider.Type() == e2e.Local {
-		nomostest.InitGitRepos(nt, newRepos...)
+		if err := nomostest.InitGitRepos(nt, newRepos...); err != nil {
+			nt.T.Fatal(err)
```

Comment:
Fatal -> Log ?

Reply:
This is a bit of a special case since it's done in a test case. I haven't looked too closely at this test but I expect the test would not be able to recover if this fails. The PortForwarder on the other hand will re-run the callback asynchronously whenever the port forward is restarted (e.g. if the Deployment's Pod is deleted/recreated)